app/controllers/idv/agreement_controller.rb

```diff
@@ -35,6 +35,7 @@ def update
 
         if IdentityConfig.store.in_person_proofing_opt_in_enabled &&
            IdentityConfig.store.in_person_proofing_enabled
+          #puts "*** #{idv_session.service_provider.inspect}"
```

Comment:
Logically we should also check whether the SP has the flag enabled. It looks like `step_info` correctly keeps us from getting there, by magic? Need to dig more into this. It might also not hurt to just add the conditional here for clarify?